lang/main-te.json

```diff
@@ -54,189 +54,189 @@
-        "AUTHFAIL": "Authentication failed",
-        "CONNECTED": "Connected",
-        "CONNECTING": "Connecting",
-        "CONNFAIL": "Connection failed",
-        "DISCONNECTED": "Disconnected",
-        "DISCONNECTING": "Disconnecting",
-        "ERROR": "Error",
-        "FETCH_SESSION_ID": "Obtaining session-id...",
-        "GET_SESSION_ID_ERROR": "Get session-id error: {{code}}",
-        "GOT_SESSION_ID": "Obtaining session-id... Done",
-        "LOW_BANDWIDTH": "Video for {{displayName}} has been turned off to save bandwidth"
+        "ATTACHED": "జోడించబడింది",
+        "AUTHENTICATING": "అధీకరిస్తూన్నది",
+        "AUTHFAIL": "అధీకరణ విఫలమైంది",
+        "CONNECTED": "అనుసంధానమైంది",
+        "CONNECTING": "అనుసంధానమవుతూన్నది",
+        "CONNFAIL": "అనుసంధానం విఫలమైంది",
+        "DISCONNECTED": "అనుసంధానం కాలేదు",
+        "DISCONNECTING": "అనుసంధానం తెంచుతున్నది",
+        "ERROR": "తప్పిదం",
+        "FETCH_SESSION_ID": "Session-id తెస్తూన్నది...",
+        "GET_SESSION_ID_ERROR": "Session-id పొందడంలో తప్పిదం: {{code}}",
+        "GOT_SESSION_ID": "Session-id తెస్తూన్నది... పూర్తయింది",
+        "LOW_BANDWIDTH": "బ్యాండ్‌విడ్త్ ఆదా చేయడం కోసం {{displayName}} నుండి వీడియో ఆపివేయబడింది"
     },
     "connectionindicator": {
         "address": "చిరునామా:",
         "audio_ssrc": "Audio SSRC:",
-        "bandwidth": "Estimated bandwidth:",
-        "bitrate": "Bitrate:",
-        "bridgeCount": "Server count: ",
-        "codecs": "Codecs (A/V): ",
-        "connectedTo": "Connected to:",
+        "bandwidth": "అంచనావేసిన బ్యాండ్‌విడ్త్:",
+        "bitrate": "బిట్‌రేటు:",
+        "bridgeCount": "సేవికల సంఖ్య: ",
+        "codecs": "కోడెక్స్ (A/V): ",
+        "connectedTo": "దీనికి అనుసంధానమైంది:",
         "e2e_rtt": "E2E RTT:",
-        "framerate": "Frame rate:",
+        "framerate": "ఫ్రేమ్ రేటు:",
         "less": "తక్కువ చూపించు",
         "localaddress": "స్థానిక చిరునామా:",
         "localaddress_plural": "స్థానిక చిరునామాలు:",
-        "localport": "Local port:",
-        "localport_plural": "Local ports:",
-        "maxEnabledResolution": "send max",
+        "localport": "స్థానిక పోర్టు:",
+        "localport_plural": "స్థానిక పోర్టులు:",
+        "maxEnabledResolution": "గరిష్ఠం పంపించు",
         "more": "ఎక్కువ చూపించు",
-        "packetloss": "Packet loss:",
+        "packetloss": "ప్యాకెట్ నష్టం:",
         "quality": {
             "good": "బాగుంది",
             "inactive": "అచేతనం",
             "lost": "పోయింది",
             "nonoptimal": "అంతబాలేదు",
             "poor": "బాలేదు"
         },
-        "remoteaddress": "Remote address:",
-        "remoteaddress_plural": "Remote addresses:",
-        "remoteport": "Remote port:",
-        "remoteport_plural": "Remote ports:",
-        "resolution": "Resolution:",
-        "savelogs": "Save logs",
-        "participant_id": "Participant id:",
-        "status": "Connection:",
+        "remoteaddress": "సుదూర చిరునామా:",
```

Comment:
Chirunama makes it sound like it is a postal address rather than the IP address.

Reply:
Yea, I think we are overloading the word just as in English. It is already in use in localizations of Firefox, MediaWiki, WordPress, etc.

I think context makes it clear to the user.

Thank you for your attention.